packages/api/config/lambdas.yml

```diff
@@ -89,6 +89,27 @@ kinesisConsumer:
       function: Ref
       value: ProvidersTableDynamoDB
     bucket: '{{system_bucket}}'
+    FallbackTopicArn:
```

Comment:
This seems a little hard to configure, where to document?  Is there a better way?